### ISSUE TITLE
Yandex browser mobile Android supporting

### ISSUE THREAD

**Issue by NotBadBit** [AI-Anywhere/AI-Anywhere#111]
Please, add possibility of using extension on Yandex mobile browser. For now it's impossible to install there:
An error has occurred
New Bing Anywhere (Bing Chat GPT-4) (extension ID "hceobhjokpdbogjkplmfjeomkeckkngi") несовместимо с мобильным Яндекс Браузером.
![IMG_20230717_233001](https://github.com/haozi/New-Bing-Anywhere/assets/53388652/cbda5955-5147-4f7c-adc3-e1dd85e60495)

**Comment by haozi** [comment]
Android？

**Comment by NotBadBit** [comment]
Yes, Yandex Browser on Android. It supports ≈90% of Chrome extensions (based on my experience of using), but not this one, unfortunately

**Comment by NotBadBit** [comment]
> Let me conduct some research before making a decision.

Possible hint: extension uBlock Origin displays the same error when trying to install the extension to the Yandex mobile browser, but developers version of it, uBlock Origin development build, can be installed and works fine on Yandex Mobile browser. Perhaps the same method will be suitable for our case 🤔
